src/test/java/com/hazelcast/eureka/one/HazelcastClientTestCase.java

```diff
@@ -64,6 +64,11 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+/**
+ * TODO: Fix this tests. They pretend to test Hazelcast Client with Eureka discovery, but
+ * that's not what they really do. One method tests Hazelcast Client with multicast and
+ * the others test Hazelcast member discovery (not Hazelcast Client!).
+ */
```

Comment:
I suggest to create a github issue. It is easier to track.

Reply:
Yeah, I've created a GH issue: https://github.com/hazelcast/hazelcast-eureka/issues/31

I'd still keep the comment here, since it took me some time while reading this class to realize what I wrote, so it may save time for others.